handlers.go

```diff
@@ -215,6 +216,7 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 	cols.POST("/create", withUser(s.handleCreateCollection))
 	cols.POST("/add-content", withUser(s.handleAddContentsToCollection))
 	cols.GET("/content/:coluuid", withUser(s.handleGetCollectionContents))
+	cols.POST("/commit/:coluuid", withUser(s.handleCommitCollection))
```

Comment:
Can you change this to `/:coluuid/commit` to be more restful? One thing I hope we do in the future is standardizing the endpoints to be restful and consistent. it will be nice going forward, that new endpoints are restful.